test/integration/shared_vpc_no_subnets/inspec.yml

```
@@ -0,0 +1,34 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: shared_vpc_no_subnets
+attributes:
+  - name: project_id
+    required: true
+    type: string
+
+  - name: project_number
+    required: true
+    type: string
+
+  - name: service_account_email
+    required: true
+    type: string
+
+  - name: shared_vpc
+    required: false
```

Attributes should be required to ensure that Terraform outputs match in naming.

```suggestion
    required: true
```